modules/setting/mailer_test.go

```diff
@@ -29,7 +31,14 @@ func Test_loadMailerFrom(t *testing.T) {
 			cfg, _ := NewConfigProviderFromData("")
 			sec := cfg.Section("mailer")
 			sec.NewKey("ENABLED", "true")
-			sec.NewKey("HOST", host)
+			if strings.Contains(host, ":") {
+				addr, port, err := net.SplitHostPort(host)
+				assert.NoError(t, err)
+				sec.NewKey("SMTP_ADDR", addr)
+				sec.NewKey("SMTP_PORT", port)
+			} else {
+				sec.NewKey("SMTP_ADDR", host)
+			}
```

Comment:
Why the tests are changed? Does it break?

Reply:
The test was test the legacy configurations. Now the fallback code was removed. So the test was changed to test the new configurations.